system/web/Renderer.cfc

```diff
@@ -207,7 +207,7 @@ component
 				arguments.args.append( explicitView.args, false );
 
 				// clear the explicit view now that it has been used
-				setExplicitView( {} );
+				setExplicitView( "", "", {} );
```

Comment:
I say, let's just remove it: `getRequestContext().removePrivateValue( "_explicitView" )`